examples/go-xdp-counter/kubernetes-deployment/go-xdp-counter.yaml

```diff
@@ -60,7 +60,7 @@ spec:
         effect: NoSchedule
       containers:
       - name: go-xdp-counter
-        image: quay.io/billy99/gocounter:latest
+        image: quay.io/billy99/go-xdp-counter:latest
```

Comment:
@astoycos Do we want a copy of go-xdp-counter and go-tc-counter (userspace containers) in quay.io/bpfd/?